src/python/datastore/data_handler.py

```diff
@@ -1181,15 +1164,13 @@ def create_user_uploaded_testcase(key,
   # Create the job to analyze the testcase.
   tasks.add_task('analyze', testcase_id, job_type, queue)
 
-  if testcase.bug_information:
-    issue = get_issue_for_testcase(testcase)
-    if issue:
-      report_url = TESTCASE_REPORT_URL.format(
-          domain=get_domain(), testcase_id=testcase_id)
-      issue.dirty = True
-      issue.comment = ('ClusterFuzz is analyzing your testcase. '
-                       'Developers can follow the progress at %s.' % report_url)
-      issue.save()
+  issue = issue_tracker_utils.get_issue_for_testcase(testcase)
+  if issue:
+    report_url = TESTCASE_REPORT_URL.format(
+        domain=get_domain(), testcase_id=testcase_id)
+    comment = ('ClusterFuzz is analyzing your testcase. '
+               'Developers can follow the progress at %s.' % report_url)
+    issue.save(new_comment=comment)
```

Comment:
Note that this function is called from two places
- directly from appengine for one testcase upload
- from unpack task with multiple testcases. i am not expecting someone to upload multiple testcases for a single bug. since this usecase is not common, we can just leave code as-is and not need to move to cleanup cron.